src/vcu/inc/controls.h

```diff
@@ -14,6 +14,15 @@
 #define MAX_ACCEL_VAL 1000
 #define MIN_ACCEL_VAL 0
 
+// Launch control contants
+// After wheel speed crosses this threshold, start using slip controller
+#define LC_WS_THRESH 45000
+#define LC_cGR 347 // Gear ratio times 100
+#define LC_ACCEL_BEGIN 950 // 95%
+#define LC_ACCEL_RELEASE 50 // 5%
+#define LC_BRAKE_BEGIN 100 // We want a lower threshold
```

Comment:
Do we still want this comment?

Reply:
Sorry, this comment was to indicate that we want lower threshold than normal. I can fix it to make it clearer